kata-deploy/scripts/kata-deploy.sh

```diff
@@ -57,10 +57,18 @@ function configure_crio() {
 	# backup the CRIO.conf only if a backup doesn't already exist (don't override original)
 	cp -n "$crio_conf_file" "$crio_conf_file_backup"
 
-	cat <<EOT | tee -a "$crio_conf_file"
+	grep -q '^\[crio.runtime.runtimes.kata-qemu\]' $crio_conf_file \
+	&& echo "Configuration exists for crio.runtime.runtimes.kata-qemu, overwriting" \
+	&& sed -i '/\[crio.runtime.runtimes.kata-qemu\]/,+1s/runtime_path.*/runtime_path = "\/opt\/kata\/bin\/kata-qemu"/' $crio_conf_file \
```

Comment:
Nits:

- Now you've got three refs to `crio.runtime.runtimes.kata-qemu`, I'd be highly tempted to turn this into a variable.
- You can simplify the sed by using `sed -i "!from!to!"` to avoid having to escape the path slashes. But in fact, why not create a variable for the path so you can just do `sed -i "/blah/$path/"`.

Reply:
@jodh-intel Done!